config/intelfocl_sample.json

```diff
@@ -0,0 +1,13 @@
+{
+  "TARGET" : "intelfocl",
+  "HW_VER" : "0.0.1",
+  "LOG_INP_WIDTH" : 3,
```

Comment:
Since the types are hard coded as 
```c
typedef char            inp_T;
typedef char            wgt_T;
```
, these config variables (e.g. `LOG_INP_WIDTH`) wouldn't actually reflect the hardware design, right?

Reply:
Yes. LOG_INP_WIDTH and LOG_WGT_WIDTH are fixed at 3 in the current version.

Reply:
In this case, we would lose the property of being "versatile" in VTA - the Versatile Tensor Accelerator. I think we need to at least warn users that these options are not actually applicable in the proposed context, or we might need to find a way to define the types accordingly with the given vta_config. @tmoreau89 what's your opinion?

Reply:
Agreed! I think for this version we will try to assert an Error if user set unsupported LOG_INP_WIDTH and LOG_WGT_WIDTH settings. And in future updates, we will enhance the code to support 16-bit and 32-bit inputs and width.

Reply:
Thanks as a resolution I think we can include assertion checks. Thanks for the catch @liangfu